### DISCUSSION TITLE
Differences between Signals and Jotai

### DISCUSSION THREAD

**Discussion by loganvolkers** [preactjs/signals#511] [General]
I'm curious about the differences between Signals and Jotai from those that have used both, or from the contributors and maintainer's perspectives. Looking for a concrete list of differences in approaches, pros and cons. There seem to be a lot of similarities.

For example, both seem to support "computed", only jotai support lifecycle events (i.e. `onMount`) and only signals support a mutation-style API.

Inspired to open this discussion based on a [Recoil vs Jotai discussion](https://github.com/pmndrs/jotai/discussions/849)

**Comment by rschristian** [comment]
The _Signal Boosting_ blog article goes into the technical details a fair bit, see: https://preactjs.com/blog/signal-boosting#implementation-notes
